web-wallet/src/lib/components/Transactions/Transactions.svelte

```diff
@@ -91,20 +96,51 @@
 								data-tooltip-place="top"
 							/>
 						</dd>
-					{/if}
-				</dl>
-			{/each}
-		{:else}
-			<div class="transactions-list__empty">
-				<Icon path={mdiContain} size="large"/>
-				<p>You have no transaction history</p>
-			</div>
-		{/if}
-	</div>
+						{#if transaction.direction === "Out"}
+							<dt class="transactions-list__term">Fee</dt>
+							<dd class="transactions-list__datum">
+								{feeFormatter(transaction.fee)}
+								<Icon
+									className="transactions-list__icon"
+									path={logo}
+									data-tooltip-id="transactions-tooltip"
+									data-tooltip-text="DUSK"
+									data-tooltip-place="top"
+								/>
+							</dd>
+						{/if}
+					</dl>
+				{/each}
+			{:else}
+				<div class="transactions-list__empty">
+					<Icon path={mdiContain} size="large"/>
+					<p>You have no transaction history</p>
+				</div>
+			{/if}
+		</div>
+
+	{:catch e}
+		<Card heading="Error getting transactions">
```

Comment:
As you're refactoring: why don't you switch to `ErrorDetails` to have a single way we show errors?

Reply:
I was focused on the actual task (fixing the design), rather than refactoring the whole component, so haven't looked at the code that was within the component.

I think this is a good suggestion and have already touch a bit on this topic in issue #1321.